apps/src/lib/client/eth_bridge/validator_set.rs

```diff
@@ -0,0 +1,46 @@
+use data_encoding::HEXLOWER;
+use namada::ledger::queries::RPC;
+
+use crate::cli::args;
+use crate::facade::tendermint_rpc::HttpClient;
+
+/// Query an ABI encoding of the validator set to be installed
+/// at the given epoch, and its associated proof.
+pub async fn query_validator_set_update_proof(args: args::ValidatorSetProof) {
+    let client = HttpClient::new(args.query.ledger_address).unwrap();
+
+    let epoch = if let Some(epoch) = args.epoch {
+        epoch
+    } else {
+        RPC.shell().epoch(&client).await.unwrap().next()
+    };
+
+    let encoded_proof = RPC
+        .shell()
+        .eth_bridge()
+        .read_valset_upd_proof(&client, &epoch)
+        .await
+        .unwrap();
+
+    println!("0x{}", HEXLOWER.encode(encoded_proof.as_ref()));
+}
+
+/// Query an ABI encoding of the validator set at a given epoch.
+pub async fn query_validator_set_args(args: args::ActiveValidatorSet) {
+    let client = HttpClient::new(args.query.ledger_address).unwrap();
+
+    let epoch = if let Some(epoch) = args.epoch {
+        epoch
+    } else {
+        RPC.shell().epoch(&client).await.unwrap()
+    };
+
+    let encoded_validator_set_args = RPC
```

Comment:
I'm not sure about `_args` in this variable and the function name

Reply:
it's just reflecting the name of the equivalent type that lives in Solidity:
https://github.com/anoma/ethereum-bridge/blob/c6389e5d8aeae0eb24d33dd83836c9ff4aace6a6/contracts/interface/ICommon.sol#L5-L9